README.md

```diff
@@ -77,3 +77,10 @@ The development environment assumes you have the following prerequisites:
 - If iterating on client/server Lua make your change and then `ensure new-phone-who-dis` in the fivem server terminal
 - If iteration on UI changes run the project with `yarn watch` which will monitor the JavaScript project and rebuild when you make code changes and then `ensure new-phone-who-dis` after making your change
 - Avoid commiting `index.html` as the development version overrides the production version
+
+#### Important Note
+
+By installing NPWD you agree to the use of the following diagnostic package, Sentry, (in use within the React portion of NPWD), that automatically
```

Comment:
What is this note?

Reply:
Privacy notice, half GDPR compliant-ish. Basically informs users that metrics are being sent regarding user sessions.